### ISSUE TITLE
no tags found when use ebpf

### ISSUE THREAD

**Issue by sunnoy** [pyroscope-io/pyroscope#1692]
i flow doc install ebpf in my k3s cluster

```bash
helm repo add pyroscope-io https://pyroscope-io.github.io/helm-chart

 helm install pyroscope-ebpf pyroscope-io/pyroscope-ebpf
helm install my-release pyroscope-io/pyroscope
```

but there ware no tags in dashbord

<img width="588" alt="image" src="https://user-images.githubusercontent.com/19342276/200766825-5c9f4e62-9dd9-45d4-b805-9410593a9701.png">

**Comment by korniltsev** [comment]
Can you share what container runtime do you use?
One way to find it is to run 

`kubectl get nodes -o wide` 
https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use/

**Comment by korniltsev** [comment]
Also could you please share cgroup name of any pod, or example from  /proc/1/cgroup for any pod.

**Comment by sunnoy** [comment]
> Can you share what container runtime do you use? One way to find it is to run
> 
> `kubectl get nodes -o wide` https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use/

k3s-1   Ready    control-plane,master   19d   v1.25.2+k3s1   172.31.105.101   <none>        Alibaba Cloud Linux 3 (Soaring Falcon)   5.10.134-12.al8.x86_64   containerd://1.6.8-k3s1

**Comment by korniltsev** [comment]
will be fixed in #1719
